CROSSSECTOR/Consumption.Point/dataModel.ConsumptionPoint/ADOPTERS.yaml

```diff
@@ -1,10 +1,10 @@
-description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
-currentAdopters:
--
- adopter: 
- description: 
- mail: 
- organization: 
- project: 
- comments: 
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```

Comment:
remove templated contents and use the proper name for the data model.

Reply:
Added new data for ADOPTERS.yaml

Reply:
I'll fix it the minor issues here